app/src/main/java/com/xmartlabs/template/common/exeption/EntityNotFoundException.java

```diff
@@ -1,7 +1,7 @@
 package com.xmartlabs.template.common.exeption;
 
 /**
- * Created by mirland on 18/01/17.
+ * To be thrown if an entity cannot be recovered from the database
```

Comment:
I left the top class javadocs that way because I haven't seen them inline, but I can change them if that's preferred